projects/core/src/lib/components/toolbar/clicked-coordinates/clicked-coordinates.component.ts

```diff
@@ -0,0 +1,85 @@
+import { ChangeDetectionStrategy, Component, OnDestroy, OnInit } from '@angular/core';
+import { concatMap, filter, Observable, Subject, takeUntil, tap } from 'rxjs';
+import { MapClickToolConfigModel, MapClickToolModel, MapService, ToolTypeEnum } from '@tailormap-viewer/map';
+import { MatSnackBar } from '@angular/material/snack-bar';
+import { Clipboard } from '@angular/cdk/clipboard';
+import { $localize } from '@angular/localize/init';
+import { Store } from '@ngrx/store';
+import { isActiveToolbarTool } from '../state/toolbar.selectors';
+import { registerTool, toggleTool } from '../state/toolbar.actions';
+import { ToolbarComponentEnum } from '../models/toolbar-component.enum';
+
+@Component({
+  selector: 'tm-clicked-coordinates',
+  templateUrl: './clicked-coordinates.component.html',
+  styleUrls: ['./clicked-coordinates.component.css'],
+  changeDetection: ChangeDetectionStrategy.OnPush,
+})
+export class ClickedCoordinatesComponent implements OnInit, OnDestroy {
+
+  private destroyed = new Subject();
+  public toolActive$: Observable<boolean>;
+
+  constructor(
+    private store$: Store,
+    private mapService: MapService,
+    private snackBar: MatSnackBar,
+    private clipboard: Clipboard,
+  ) {
+    this.toolActive$ = this.store$.select(isActiveToolbarTool(ToolbarComponentEnum.SELECT_COORDINATES));
+  }
+
+  public ngOnInit(): void {
+    this.mapService.createTool$<MapClickToolModel, MapClickToolConfigModel>({
+      type: ToolTypeEnum.MapClick,
+    })
+      .pipe(
+        takeUntil(this.destroyed),
+        filter(Boolean),
+        tap(clickTool => {
+          this.store$.dispatch(registerTool({ tool: { id: ToolbarComponentEnum.SELECT_COORDINATES, mapToolId: clickTool.id }}));
+        }),
+        concatMap(clickTool => clickTool.mapClick$),
+      )
+      .subscribe(mapClick => {
+        this.handleMapClick(mapClick);
+      });
+  }
+
+  public ngOnDestroy() {
+    this.destroyed.next(null);
+    this.destroyed.complete();
+  }
+
+  public toggle() {
+    this.store$.dispatch(toggleTool({ tool: ToolbarComponentEnum.SELECT_COORDINATES }));
+  }
+
+  private handleMapClick(evt: { mapCoordinates: [number, number]; mouseCoordinates: [number, number] }) {
+    let decimals = 4;
+    this.mapService.getUnitsOfMeasure$().subscribe(uom => {
+      switch (uom) {
+        case 'm':
+          decimals = 2;
+          break;
+        case 'ft':
+        case 'us-ft':
+          decimals = 3;
+          break;
+        case 'degrees':
+        default:
+          decimals = 4;
+      }
+    });
+
+    const coordinatesMsg = $localize`Selected coordinates: ${evt.mapCoordinates[0].toFixed(decimals)}, ${evt.mapCoordinates[1].toFixed(decimals)}`;
+
+    this.snackBar.open(coordinatesMsg, $localize`Copy`).onAction().subscribe(() => {
+      if (this.clipboard.copy(coordinatesMsg)) {
```

Comment:
Maybe just copy the coordinates so they are easier to copy/past in other application without having to remove the 'Selected coordinates' part.

Reply:
accessibility wise not a good idea, as the message will be announced as just the 2 numbers, also in general most GIS applications receiving a coordinate pair want them as 2 separate values (note that for some CRS they may need to be flipped in order as well). 

It might be better to provide a WKT

Reply:
I meant only the clipboard action, so `this.clipboard.copy(`${evt.mapCoordinates[0].toFixed(decimals)}, ${evt.mapCoordinates[1].toFixed(decimals)}`)` but keep the snackbar message as is. This way people can just paste the numbers in other apps

Reply:
+1